frontend/src/components/VTreeView/_store/index.js

```diff
@@ -0,0 +1,27 @@
+const state = {
```

Comment:
this entire store is a mistake by me I haven't removed yet. It's not being used. So just remove this store :)